daemon/mgr/image_utils.go

```diff
@@ -16,6 +16,38 @@ import (
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+// Multierrors contains a slice of errors.
```

Comment:
i think `Multierrors` not attached to image, so we should put this code to `pkg` dir, maybe `pkg/error`, WDYT ?😆

Reply:
We can move it out to `pkg` if necessary. The `pkg/error` only contains the `Multierrors`. It is weird... 😈

Reply:
I think it's ok, some `cli` command also need the `Multierrors`, so it's proper to move it to `pkg`, WDTY?? 😆

Reply:
Could you mind to give me a example?

Reply:
I can move it to the `pkg/utils` in case that some one wants to use.